src/foam/u2/layout/Cards.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.layout',
+  name: 'Cards',
+  extends: 'foam.u2.Element',
+  css: `
+    ^ {
+      display: flex;
+      justify-content: flex-start;
+    }
+  `,
+
+  properties: [
+    {
+      class: 'String',
+      name: 'defaultMargin',
+      value: '8px'
+    }
+  ],
+
+  methods: [
+    function initE() {
+      this.SUPER();
+      this.addClass(this.myClass());
+    },
+
+    function start(spec, args, slot) {
```

Comment:
Let's avoid doing this and just give Card a default margin and anybody rendering them that wants to override the margin can do it via u2. If we find ourselves doing this frequently we'll find a way to conveniently do that but until then I'd rather keep the configuration we expose here to a minimum

Reply:
Done.